docs/authorization_userguide.md

```diff
@@ -60,5 +60,6 @@ Currently, Supported statements are as follows:
 
 ## limitations
 
+- Not supported with hive as datasource
```

Comment:
The description is not accurate.
It's not only unsupported but also can't work with other data sources if you turn on auth mode

Reply:
done